### ISSUE TITLE
Track and display pids in spfs runtime info

### ISSUE THREAD

**Issue by dcookspi** [spkenv/spk#1059]
**Is your feature request related to a problem? Please describe.**

Sometimes we see a lot of `stopping` runtimes on a host when using `spfs runtime list`.  Particularly on renderfam hosts that launch many subprocesses inside each /spfs runtime.

`spfs-monitor` is running for those runtimes, but the owner process and original /spfs mounstspawn process are not running (according to `spfs runtime list`). But some other processes are still running in/using that /spfs mount. 

Unless you have root permissions and wade through the /proc/fs looking for the mounts you can't tell what processes are still keeping that /spfs alive.

**Describe the solution you'd like**
Have `spfs runtime list`, or `spfs runtime info`, show which pid/s are still using a runtime's /spfs so we can see which processes `spfs-monitor` and /spfs are waiting on and see what's keeping the mount active.

**Additional context**
This probably involves keeping more pid info in the runtime and having `spfs-monitor` update it periodically as processes die/start. That needs some discusson and sanity checking.

**Comment by rydrman** [comment]
From the meeting today:
- adding at least one PID to the runtime to represent why the runtime is still alive
- would be great if we could try at least a little to ensure it's a useful one - maybe prefer numbers that still exist across time? Not critical but nice to have.
